src/Compilers/CSharp/Portable/Binder/ForEachLoopBinder.cs

```diff
@@ -1535,7 +1535,7 @@ private MethodArgumentInfo FindForEachPatternMethodViaExtension(SyntaxNode synta
                 returnRefKind: default,
                 returnType: null,
                 ref extensionUseSiteInfo,
-                acceptOnlyMethods: true); // Tracked by https://github.com/dotnet/roslyn/issues/76130 : Test effect of acceptOnlyMethods value
```

Comment:
:memo: this is not observable. If we got an extension property returned in the result here, the way we look at the overload resolution result below would ignore it

Reply:
In that scenario, `methodGroupResolutionResult.OverloadResolutionResult` is null.
The found property is stored in `methodGroupResolutionResult.OtherSymbol`.
A relevant test is `ExtensionMemberLookup_PatternBased_ForEach_DelegateTypeProperty`